Form-Controls/index.html

```diff
@@ -13,15 +13,67 @@ <h1>Product Pick</h1>
+            <input type="text" id="name" name="user_name" required minlength="2" maxlength="50" />
+
+          </p>
+          <p>
+            <label for="mail">Email:</label>
+            <input type="email" id="mail" name="user_email" required/>
+          </p>
+          <p>
+            <label for="colour"> What colour should this T-shirt be? </label> <br>
+              <input type="radio" id="green" name="Tshirt_colour" value="Green">
+                <label for="green"> Green</label>
+                <input type="radio" id="red" name="Tshirt_colour" value="Red">
+                <label for="red"> Red</label>
+                <input type="radio" id="yellow" name="Tshirt_colour" value="Yellow">
+                <label for="yellow"> Yellow</label>               
+          </p>
+          <p>
+            <label for="size"> Please choose a size. </label><br>
+                <input type="radio" id="xs" name="Tshirt_size" value="XS">
+                <label for="xs"> XS</label>
+
+                <input type="radio" id="s" name="Tshirt_size" value="S">
+                <label for="s"> S</label>
+
+                <input type="radio" id="m" name="Tshirt_size" value="M">
+                <label for="m"> M </label>
+
+                <input type="radio" id="l" name="Tshirt_size" value="L">
+                <label for="l"> L</label>
+
+                <input type="radio" id="xl" name="Tshirt_size" value="XL">
+                <label for="xl"> XL</label>
+
+                <input type="radio" id="xxl" name="Tshirt_size" value="XLL">
+                <label for="xxl"> XXL</label>   
+                
+
+          </p>
+          <p>
+           <button type="submit"> SUBMIT</button>
+          </p>
         <!-- write your html here-->
         <!-- 
           try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+                this will also help you fill in your PR message later
+                Our customers already have accounts, so we know their addresses and charging details already. We don't need to collect that data. We want to confirm they are the right person, then get them to choose a colour and size.
+
+              Writing that out as a series of questions to ask yourself:
+
+              1. What is the customer's name? I must collect this data, and validate it. But what is a valid name? I must decide something.
+              2. What is the customer's email? I must make sure the email is valid. Email addresses have a consistent pattern.
+              3. What colour should this t-shirt be? I must give 3 options. How will I make sure they don't pick other colours?
+              4. What size does the customer want? I must give the following 6 options: XS, S, M, L, XL, XXL
+        -->
       </form>
     </main>
     <footer>
       <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
+      <h2>By CHRISTELLE BOTEN</h2>
```

Comment:
I don't think that heading level 2 is appropriate semantic tag here. Please think of a better tag for this case.